R-packages/evalcast/R/plot_calibration.R

```diff
@@ -127,6 +132,11 @@ format_traditional_calib_plot <- function(calib,
                                           facet_rows,
                                           facet_cols){
   
+  if (is.null(facet_rows) && is.null(facet_cols)){
+    facet_rows <- c("ahead")
```

Comment:
Do we need to facet on both by default? Since the traditional ones can plot multiple lines, I think I've seen them facet on one fewer variable then the wedge plots.

Reply:
The default `grp_vars` have three variables, so the default plots one var for rows, one for cols, and one for colours.  This is the behavior that existed originally.